From: Nicholas Mc Guire <hofrat@osadl.org>
Date: Fri, 6 Feb 2015 06:52:17 -0500
Subject: [PATCH] ceph: use msecs_to_jiffies for time conversion
Git-commit: 3563dbdd99603d4339467ddffafe3851aae2e398
Patch-mainline: v4.1
References: Fate#318586


This is only an API consolidation and should make things more readable
it replaces var * HZ / 1000 by msecs_to_jiffies(var).

Signed-off-by: Nicholas Mc Guire <hofrat@osadl.org>
Signed-off-by: Yan, Zheng <zyan@redhat.com>
Acked-by: NeilBrown <neilb@suse.com>

---
 fs/ceph/mds_client.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/fs/ceph/mds_client.c
+++ b/fs/ceph/mds_client.c
@@ -3142,7 +3142,7 @@ static void handle_lease(struct ceph_mds
 		    di->lease_renew_from &&
 		    di->lease_renew_after == 0) {
 			unsigned long duration =
-				le32_to_cpu(h->duration_ms) * HZ / 1000;
+				msecs_to_jiffies(le32_to_cpu(h->duration_ms));
 
 			di->lease_seq = seq;
 			dentry->d_time = di->lease_renew_from + duration;
